test/L1/SwapAndBridge.t.sol

```diff
@@ -0,0 +1,169 @@
+        vm.assume(value <= 1e38);
+        // Ensure that some tokens will be minted in the MockERC20LST contract.
+        vm.assume(tokensPerETH * value >= 1e18);
+
+        MockERC20LST mockLST = new MockERC20LST(tokensPerETH);
+        MockStandardBridge bridge = new MockStandardBridge();
+        SwapAndBridge swapAndBridge = new SwapAndBridge(address(bridge), address(mockLST), address(0xbeef));
+
+        uint256 expectedAmount = value * tokensPerETH / 1e18;
+
+        vm.startPrank(testAccount);
+        vm.expectRevert("Insufficient L1 tokens minted.");
+        swapAndBridge.swapAndBridgeToWithMinimumAmount{ value: value }(testAccount, expectedAmount + 1);
+        vm.stopPrank();
+    }
+
+    function test_TokensMintedOverflow(uint256 value, uint256 tokensPerETH) public {
+        // Guaranteee overflow in MockERC20LST contract.
+        vm.assume(value > 1e40);
+        vm.assume(tokensPerETH > 1e40);
+        vm.assume(value <= testBalance);
+
+        MockERC20LST mockLST = new MockERC20LST(tokensPerETH);
+        MockStandardBridge bridge = new MockStandardBridge();
+        SwapAndBridge swapAndBridge = new SwapAndBridge(address(bridge), address(mockLST), address(0xbeef));
+
+        vm.startPrank(testAccount);
+        vm.expectRevert(); // overflow panic.
+        swapAndBridge.swapAndBridgeToWithMinimumAmount{ value: value }(testAccount, tokensPerETH);
+        vm.stopPrank();
+    }
+
+    function test_InvalidMsgValue(uint256 tokensPerETH, uint256 minL1Tokens) public {
+        MockERC20LST mockLST = new MockERC20LST(tokensPerETH);
+        MockStandardBridge bridge = new MockStandardBridge();
+        SwapAndBridge swapAndBridge = new SwapAndBridge(address(bridge), address(mockLST), address(0xbeef));
+
+        vm.startPrank(testAccount);
+        vm.expectRevert("Invalid msg value.");
+        swapAndBridge.swapAndBridgeToWithMinimumAmount{ value: 0 }(testAccount, minL1Tokens);
+        vm.stopPrank();
+    }
+
+    function test_NoTokensMinted(uint256 value, uint256 tokensPerETH) public {
+        // We bound tokensPerETH in fuzzzing to avoid overflows in the MockERC20LST contract.
+        vm.assume(tokensPerETH <= 1e38);
+        vm.assume(value > 0);
+        vm.assume(value <= 1e38);
+        // Ensure that no tokens will be minted in the MockERC20LST contract.
+        vm.assume(tokensPerETH * value < 1e18);
+
+        MockERC20LST mockLST = new MockERC20LST(tokensPerETH);
+        MockStandardBridge bridge = new MockStandardBridge();
+        SwapAndBridge swapAndBridge = new SwapAndBridge(address(bridge), address(mockLST), address(0xbeef));
+
+        vm.startPrank(testAccount);
+        vm.expectRevert("No wrapped tokens minted.");
+        swapAndBridge.swapAndBridgeToWithMinimumAmount{ value: value }(testAccount, 0);
+        vm.expectRevert("No wrapped tokens minted.");
+        swapAndBridge.swapAndBridgeToWithMinimumAmount{ value: value }(testAccount, 1e18);
+        vm.stopPrank();
+    }
```

Comment:
indeed I was not testing them as it would require introducing bugs in the Mock contracts basically. 

I added the fake contracts and relative tests.